### PR TITLE
[MTM-50741] Updated screenshots for Administration > Changing settings (en + de)

### PR DESCRIPTION
**Pull request by pawel-rynarzewski-sag**

### CODE REVIEW

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 1]
```diff
@@ -478,7 +478,7 @@ Die Kontexte des Tokens werden im JSON-Format dargestellt.
```
Comment (pawel-rynarzewski-sag) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:478]: @BeateRixen, noticed that English and German user guides have different structures (some parts are in different order, or some parts are mixed together within another section's file):

Might be good to align the structures, when the updates are done (I know other team is now updating TFA and SSO sections, so don't want to mess with them now).

Comment (BeateRixen) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:478]: @pawel-rynarzewski-sag The German guides still show the 10.15 content. 10.16 is currently being translated and we will then replace the files accordingly. Changes you made manually in the German md files will be overwritten with the next translation cycle. I should have explained that, sorry. We actually only need the German images, the texts will be adapted automatically.

**Review by BeateRixen — APPROVED** [review]

**Review by jkot-softwareag — APPROVED** [review]

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md (hunk 1 above)
Comment (pawel-rynarzewski-sag) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:478]: Ah, I see, thanks for explaining. :+1:

**Comment by BeateRixen** [comment]
@pawel-rynarzewski-sag Should I merge this PR?

**Comment by pawel-rynarzewski-sag** [comment]
@BeateRixen, yes, please merge.

**Pull request merged**